deploy/esh

```diff
@@ -0,0 +1,366 @@
+#!/bin/sh
```

Comment:
Is this not a library we can depend on or something we can add to storoku? Would be good to not have to vendor this in every repo.